graphql_compiler/tests/integration_tests/integration_test_helpers.py

```diff
@@ -25,9 +27,24 @@ def sort_key(result):
     return sorted(results, key=sort_key)
 
 
+def try_convert_decimal_to_string(value):
+    """Return Decimals as string if value is a Decimal, return value otherwise."""
+    if isinstance(value, list):
+        return [try_convert_decimal_to_string(subvalue) for subvalue in value]
+    if not isinstance(value, Decimal):
+        return value
+    return str(value)
+
+
 def compile_and_run_match_query(schema, graphql_query, parameters, graph_client):
     """Compiles and runs a MATCH query against the supplied graph client."""
-    compilation_result = graphql_to_match(schema, graphql_query, parameters)
+    # OrientDB expects decimals to be passed as their string representation
```

Comment:
nit: This is slightly misleading. The MATCH code the compiler emits expects Decimals to come in as strings, since they are wrapped in a statement that causes OrientDB to parse them into Decimal types before using them.

Reply:
Corrected, thanks for the explanation.